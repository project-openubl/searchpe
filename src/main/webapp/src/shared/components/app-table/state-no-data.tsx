import React from "react";
import {
  EmptyState,
  EmptyStateBody,
  EmptyStateIcon,
  EmptyStateVariant,
  Title,
} from "@patternfly/react-core";
import { AddCircleOIcon } from "@patternfly/react-icons";

export const StateNoData: React.FC = () => {
  return (
    <EmptyState variant={EmptyStateVariant.small}>
      <EmptyStateIcon icon={AddCircleOIcon} />
      <Title headingLevel="h2" size="lg">
        No data available
      </Title>
      <EmptyStateBody>
        Create a new resource to search using this table.
      </EmptyStateBody>
    </EmptyState>
  );
};
