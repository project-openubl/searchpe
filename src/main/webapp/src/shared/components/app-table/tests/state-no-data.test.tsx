import React from "react";
import { shallow } from "enzyme";

import { StateNoData } from "../state-no-data";

describe("StateNoData", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(<StateNoData />);
    expect(wrapper).toMatchSnapshot();
  });
});
