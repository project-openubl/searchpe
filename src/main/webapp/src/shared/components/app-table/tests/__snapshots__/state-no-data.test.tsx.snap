// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StateNoData Renders without crashing 1`] = `
<EmptyState
  variant="small"
>
  <EmptyStateIcon
    icon={[Function]}
  />
  <Title
    headingLevel="h2"
    size="lg"
  >
    No data available
  </Title>
  <EmptyStateBody>
    Create a new resource to search using this table.
  </EmptyStateBody>
</EmptyState>
`;
